### PR TITLE
message sender client for wassup

### PR DESCRIPTION
**Pull request by smn**
A client for Wassup in the message sender. This means we'll not need
to use Junebug anymore for sending & receiving.

### CODE REVIEW

**Comment by smn** [comment]
@erikh360 & @KaitCrawford & @rudigiesler this is ready for review

File: message_sender/models.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Channel(models.Model):
     VUMI_TYPE = 'vumi'
     JUNEBUG_TYPE = 'junebug'
     HTTP_API_TYPE = 'http_api'
+    WASSUP_API_TYPE = 'wassup'
```
Comment (rudigiesler) [inline message_sender/models.py:15]: Should we not also add this to `CHANNEL_TYPES`, and make a migration for it?

Comment (smn) [reply message_sender/models.py:15]: good point, will add.

**Comment by smn** [comment]
@rudigiesler ready for review, has views for events and adds a serializer for inbound Wassup messages. The `InboundViewset` has a lot of branching to deal with vumi/junebug/wassup and is prime for some refactoring.

File: message_sender/factory.py [hunk 2]
```diff
@@ -153,6 +154,76 @@ def _raw_send(self, py_data):
...
+class WassupApiSenderException(Exception):
+    pass
+
+
+WASSUP_SESSIONS = {}
+
+
+class WassupApiSender(object):
+
+    default_session = requests.Session()
```
Comment (rudigiesler) [inline message_sender/factory.py]: I don't think that we use this `default_session` anywhere?

**Review by rudigiesler — COMMENTED** [review]
Just one small comment from me, looks good otherwise :)

**Pull request merged**